test/pkg/integration/integration.go

```diff
@@ -122,7 +180,7 @@ func Instrument(component ComponentType, agentName string, namespace string, cli
 		return nil, closer, err
 	}
 
-	cmd := []string{filepath.Join("/tmp", tgtFN), "-rpc-port", strconv.Itoa(localAgentPort)}
+	cmd := []string{filepath.Join("/home/gitpod/", tgtFN), "-rpc-port", strconv.Itoa(localAgentPort)}
```

Comment:
I changed it to `/home/gitpod` because copying to `/temp` folder was failing, same if i changed it to `/workspace/reponame`.
You can reproduce with `kubectl cp /some/file ws-pod:/tmp/` and then manually check the `/tmp` folder in the workspace, the file won't be there

Reply:
FYI @csweichel, `WorkspacekitLift` was already set to true by default in line 142 so the change we added today `, integration.WithWorkspacekitLift(true)` was irrelevant

Reply:
The file not being present makes sense, however with the `lift` it should work nonetheless.
I'm surprised to see that we had already used it.

What's the error message you're getting with the original code?

Reply:
This is what I found:
- Using original code and copying to `tmp` folder, command runs "successfully" with no error but the actual file is not present in `tmp` folder in the end
- Using original code and copying to '/home/gitpod/' folder works just fine
- Using original code and prepending lift to the tar command just hangs indefinitely while waiting the error group line 308
```
cmd := []string{"tar", "-xmf", "-", "-C", "/tmp"}
cmd = append([]string{"/.supervisor/workspacekit", "lift"}, cmd...)
```